api-scripts/feesfines/DataPreparation/AccountCreation/loanID.txt

```diff
@@ -0,0 +1,5158 @@
+9ded16fa-6e11-4c53-89c1-a76d62482d80
```

Comment:
loanId.txt and loanID.csv are the same. Why do you need both?

Reply:
loanID.csv is modified for JMeter script usage. and loanId.txt is for data seeding. just wanted to keep different processes separately.  Will resolve in scope of https://github.com/folio-org/perf-testing/pull/43#discussion_r551130416